src/core/ext/filters/client_channel/xds/xds_bootstrap.cc

```diff
@@ -58,23 +58,23 @@ XdsBootstrap::XdsBootstrap(grpc_slice contents, grpc_error** error)
     return;
   }
   InlinedVector<grpc_error*, 1> error_list;
-  bool seen_xds_server = false;
+  bool seen_xds_servers = false;
   bool seen_node = false;
   for (grpc_json* child = tree_->child; child != nullptr; child = child->next) {
     if (child->key == nullptr) {
       error_list.push_back(
           GRPC_ERROR_CREATE_FROM_STATIC_STRING("JSON key is null"));
-    } else if (strcmp(child->key, "xds_server") == 0) {
```

Comment:
Just to confirm: is it true that `xds_server` is no longer possible to be seen in the bootstrap file?

Reply:
Yes.  No one is using this in production yet, so we can still change it at this point in non-backward-compatible ways.